src/styles/blocks/timetable.scss

```diff
@@ -0,0 +1,49 @@
+.timetable {
+
+  &__before {
+    font-weight: 700;
+    font-size: 12px;
+    line-height: 140%;
+    color: #d12d35;
+
+    margin: 0 0 2px;
+  }
+
+  &__bottom {
+    margin-bottom: 32px;
+    text-align: left;
+  }
+
+  &__title {
+    margin-bottom: 16px;
+  }
+
+  &__hours-avenue {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-items: center;
+
+    margin-bottom: 16px;
+  }
+
+  &__hours-cloisters {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-items: center;
```

Comment:
Replace 'justify-items: center;' with 'justify-content: center;' to correctly align the items horizontally in the flex container.